integration/commands_freemonitoring_test.go

```diff
@@ -148,6 +148,12 @@ func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
 				actualStatus, ok := actual.Map()["state"]
 				require.True(t, ok)
 
+				if actualStatus == "disabled" {
+					if _, ok := actual.Map()["debug"]; ok {
+						actualStatus = actual.Map()["debug"].(bson.D).Map()["state"]
```

Comment:
Thanks for the change, great solution :hugs: This would work well :+1: I have minor suggestions.

When we read do type assertion such as `.(bson.D)`, if the type is not `bson.D` this case it it would panic. I would suggest `debug, ok := v.(bson.D)` and checking `require.True(t, ok)` to fail the test rather than panic :hugs: By the way, you could assign  `v, ok := actual.Map()["debug"]` to use variable `v` instead.

Also for the last part of `.Map()["state"]`, let's check that `state` field exists similarly to https://github.com/FerretDB/FerretDB/blob/96f304606972c59455fd2c893ccaba1db0461a88/integration/commands_freemonitoring_test.go#L148-L149

Reply:
I've made the changes, thanks for the suggestion!